### PR TITLE
146 updates for setup page

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/146)
[staging](https://650c80979fc55b0dc6a1822c--tangerine-buttercream-20c32f.netlify.app/developers/academy/zero_to_mvp/hello_weaviate/set_up)

### What's being changed:
Tweaks on the db set up page

### Type of change:
- [X] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [X] **Github action** – automated build completed without errors
- [X] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

**Comment by daveatweaviate** [comment]
@databyjp  here's another one, would you have a look when you get a moment? Thanks!

File: developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx [hunk 1]
```diff
@@ -18,94 +18,103 @@ import ReactPlayer from 'react-player/lazy'
 
 ## <i class="fa-solid fa-square-chevron-right"></i>&nbsp;&nbsp;Options for running Weaviate
 
-You can run Weaviate as a managed service (Weaviate Cloud Services, or WCS), or manage your own instances using Docker, Kubernetes, or Embedded Weaviate.
+Weaviate Cloud Services (WCS) can run Weaviate for you as a managed service. You can also manage your own Weavaite instances using Docker, Kubernetes, or Embedded Weaviate.
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx:21]: I think we generally talk about WCS as a product not as an autonomous entity. Something else for the style guide perhaps.

I see your first `Weavaite` typo! You're definitely one of us now.

File: developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx [hunk 2]
```diff
@@ -18,94 +18,103 @@ import ReactPlayer from 'react-player/lazy'
...
 
 ### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;WCS (Recommended)
 
 [WCS, or Weaviate Cloud Services](https://console.weaviate.cloud/), is a managed SaaS service that requires no maintenance at your end.
 
-As it is managed by Weaviate (the company - the software is not sentient... *yet*), it is the fastest way to create a new instance of Weaviate and requires the least amount of effort for the users.
+As it is managed by Weaviate (the company - the software is not sentient... *yet*). WCS is the fastest way to create a new instance of Weaviate and requires the least amount of effort for users.
 
-Weaviate instances on WCS come pre-configured for convenience. They include a number of Weaviate "modules" by default, as well as built-in support for user authentication for security.
+The Weaviate instances in WCS are pre-configured for your convenience. They include a number of Weaviate modules by default. They also have built-in support for user authentication.
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx:31]: I think `on` is preferable for the same reason as below, but I don't feel very strongly about it tbh.

File: developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx [hunk 3]
```diff
@@ -18,94 +18,103 @@ import ReactPlayer from 'react-player/lazy'
...
-### <i class="fa-solid fa-code"></i>&nbsp;&nbsp;Create a Weaviate Cluster
+1. If you don't have a WCS account, click on the "Register" button to create a new account.
+
+1. Click on "Sign in with the Weaviate Cloud Services" and sign in with your WCS username and password.
+
+### <i class="fa-solid fa-arrow-right-from-bracket"></i>&nbsp;&nbsp; Create a Weaviate Cluster
 
 To create a new Weaviate Cluster, click the "Create cluster" button.
 
+[comment]: # (The UI has changed slightly, we should update or drop this image)
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/40_set_up.mdx:65]: Yeah I am for fewer images.

**Comment by databyjp** [comment]
Thanks @daveatweaviate - these are great; and I defer to you on images. I would prefer to remove the UI images as you highlighted, and replace them with written instructions where applicable.

**Pull request merged**